Specs/Core/BoxGeometrySpec.js

```diff
@@ -84,6 +86,22 @@ defineSuite([
         expect(m.indices.length).toEqual(12 * 3);
     });
 
+    it('fromAxisAlignedBoundingBox throws with no boundingBox', function() {
+        expect(function() {
+            return BoxGeometry.fromAxisAlignedBoundingBox(undefined);
+        }).toThrowDeveloperError();
+    });
+
+    it('fromAxisAlignedBoundingBox', function() {
+        var min = new Cartesian3(-1, -2, -3);
+        var max = new Cartesian3(1, 2, 3);
+        var m = BoxGeometry.createGeometry(BoxGeometry.fromAxisAlignedBoundingBox(
```

Comment:
No need use to use `BoxGeometry` here.  This is just a unit test - not a system test - let's just construct a `BoxGeometry` and check that the properties are set.